### PR TITLE
Add documentation for cache control headers on cloud-backed media

### PR DESCRIPTION
**Pull request by CamronStaley**
## What changes are proposed in this pull request?

Updating documentation to inform customers about how to enable browser caching for our supported cloud media types.

https://voxel51.atlassian.net/browse/TEAMS-3180

## How is this patch tested? If it is not, please explain why.

n/a

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [x] Documentation: FiftyOne documentation changes
-   [ ] Other

## Summary by CodeRabbit

- **Documentation**
	- Added a new section on "Browser Caching" to improve guidance on optimizing performance for cloud-backed media.
	- Included specific recommendations for configuring cache-control headers for various cloud platforms, enhancing data handling efficiency for users.

### CODE REVIEW

**Review by sashankaryal — CHANGES_REQUESTED** [review]
🚀 🚀 🚀 
looking good, requesting changes for adding a line or two that describes each provider's defaults

File: docs/source/teams/installation.rst
Comment (CamronStaley) [reply docs/source/teams/installation.rst:184]: +1 - for the other comments I can include the default but I feel like in general we should just tell them to do it instead of hoping that the browser or storage solution handles it properly. Like Stuart mentioned on the ticket some of these "defaults" change based on how you've setup your storage permissions so it's possible our default is different from theirs just based on how we have configured our data to begin with. I'm afraid to say (as an example) "S3 doesn't provide cache headers by default" but if their bucket has X configuration it starts some default value.

Comment (sashankaryal) [reply docs/source/teams/installation.rst]: Valid. I'm not aware of anything you can configure bucket-level that affects that particular metadata in either S3 or GCP - which is why I insisted on including it - but I defer to you - what you're saying makes sense.

Comment (CamronStaley) [reply docs/source/teams/installation.rst]: I was looking around online and seems like GCP is the only one providing a default value (added these in a commit) but probably best to also strongly recommend them to configure it for themselves instead of taking the gamble.

**Review by sashankaryal — APPROVED** [review]

**Comment by sashankaryal** [comment]
LGTM! 🎸

**Comment by benjaminpkane** [comment]
Be sure the docs formatting looks correct before merging. Link [here](https://github.com/voxel51/fiftyone/actions/runs/10166212529/artifacts/1756486986) 😄

**Comment by CamronStaley** [comment]
@benjaminpkane downloaded and reviewed locally it looks correct merging now!

**Pull request merged**
